.github/workflows/gh-pages-releases.yml

```diff
@@ -0,0 +1,40 @@
+# See: https://github.com/llvm/torch-mlir/issues/1374
+name: Publish releases page
+
+on:
+  schedule:
+    - cron: '0 * * * *'
+
+  workflow_dispatch:
+
+jobs:
+  scrape_and_publish_releases:
+    name: "Scrape and publish releases"
+    runs-on: ubuntu-20.04
+
+    # Don't run this in everyone's forks.
+    if: github.repository == 'llvm/torch-mlir'
+
+    steps:
+      - name: Checking out repository
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.WORKFLOW_INVOCATION_TOKEN }}
+      - name: Run scrape releases script
+        run: python ./tools/scrape_releases.py llvm torch-mlir > /tmp/index.html
+        shell: bash
+      - run: git fetch --all
+      - run: git switch github-pages
+      - run: git config --global user.email "none@none.com"
+      - run: git config --global user.name "torch-mlir"
+      - run: mv /tmp/index.html index.html
```

Comment:
nit: "index.html" is kind of like a special "main" IIRC for websites ([link](https://www.pcmag.com/encyclopedia/term/indexhtml) found through quick googling). Let's use a slightly more verbose name `package_index.html` to dodge that special name.

Reply:
I'm suddenly feeling very old remembering when the transition happened to no longer having html extensions in URLs and realizing that was a while ago haha.  I've moved everything to `package-index` in an index file there so we still get the nice URL.  (Should be visible at https://llvm.github.io/torch-mlir/package-index/)

Reply:
nice :)

minor nit, can you add some newlines/<br>'s so that when opening in the browser it is one link per line. Sometimes I have looked at PyTorch's equivalent of this page in the browser for various reasons.

Reply:
Yup!  Good idea, thanks.  Might as well make it readable for humans too, since I am one.